src/client/instant-meilisearch-client.ts

```diff
@@ -103,11 +98,13 @@ export function instantMeiliSearch(
           // Adapt the Meilisearch response to a compliant instantsearch.js response
           const adaptedSearchResponse = adaptSearchResponse<T>(
             searchResponse,
-            searchContext
+            searchContext,
+            initialFacetDistribution[searchRequest.indexName]
```

Comment:
removed `initialFacetDistribution` from the `searchContext` as it is not correlated to the current search request but as value for the whole instance